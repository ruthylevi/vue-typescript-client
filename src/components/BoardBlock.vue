<template>
  <v-card class="elevation-2">
    <v-card-title class="headline" v-text="date"></v-card-title>
    <v-card-text class="text-left" v-text="desc"></v-card-text>
    <v-img v-for="pic in img" :key="pic" height="350" :src="pic"></v-img>
  </v-card>
</template>

<script>
export default {
  props: {
    date: String,
    desc: String,
    img: Array,
  },
};
</script>
