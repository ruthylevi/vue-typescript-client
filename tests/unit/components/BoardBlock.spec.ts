import Vue from 'vue';
import Vuetify from 'vuetify';
import VueRouter from 'vue-router';
import { shallowMount } from '@vue/test-utils';
import BoardBlock from '@/components/BoardBlock.vue';

Vue.use(Vuetify);
Vue.use(VueRouter);

const wrapper = shallowMount(BoardBlock);

describe('BoardBlock', () => {
  test('Sanity', () => {
    expect(true).toBe(true);
  });

  test('Visible', () => {
    expect(wrapper.isVisible()).toBe(true);
  });

  test('Renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
