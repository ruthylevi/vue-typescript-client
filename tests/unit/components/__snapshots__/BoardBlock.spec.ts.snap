// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoardBlock Renders correctly 1`] = `
<v-card-stub loaderheight="4" tag="div" class="elevation-2">
  <v-card-title-stub class="headline"></v-card-title-stub>
  <v-card-text-stub class="text-left"></v-card-text-stub>
</v-card-stub>
`;
